setup.py

```diff
@@ -16,7 +16,7 @@
     license='MIT License',
     install_requires=['numpy', 'torch'],
     extras_require={
-        'test': ['pytest'],
+        'test': ['pytest', 'optuna'],
```

Comment:
Could you let me know how to add dependency for tests? I thought just adding libraries into `setup.py`, but tests failed.
Should I add them into `.flexci/test.sh` instead of `setup.py`, right?
https://github.com/dhgrs/pytorch-pfn-extras/blob/2dfbffa6b1cc1f14c2712d0c884b05fa8369d8d4/.flexci/test.sh#L5

Reply:
@kmaehashi

Reply:
Currently test dependencies must be listed in `test.sh`, but I think it's better to list them in `setup.py` instead. You can either do thatin this PR, or just add `optuna` to `test.sh` so that we will work on it in another PR.

Reply:
Thank you for your answer. Then I'm going to add a commit like
- Add `optuna` to `flexci/test.sh`
- Remove `optuna` in `setup.py` to avoid a conflict in another PR.